File: java_old/driver/Tile.java
package driver;

/**
 * All tiles should be initialized before the game begins.
 * @author Scott Sperling
 */

public class Tile {
	//Data 
	public String letter;
	public int pointValue;

	public Tile() {
		
	}
	
	/**
	 * @param l The letter of the tile
	 * @param pV The point value of the tile
	 */
	public Tile(String l, int pV) {
		letter = l;
		pointValue = pV;
	}
	
	/**
	 * Overridden function to print the letter and point value of a tile
	 */
	public String toString() {
		return letter + " " + pointValue;
	}
	
}

File: python_old/tile.py
class Tile(object):
    def __init__(self, letter, points):
        self.letter = letter
        self.points = points

    def __eq__(self, other):
        return self.letter == other.letter and self.points == other.points

    def __str__(self):
        string = self.letter + " " + str(self.points)
        return string
